### DISCUSSION TITLE
OMGlib.

### DISCUSSION THREAD

**Discussion by bhw-foss** [rabbibotton/clog#166] [General]
Dr,

Was browsing the updated projects in this months Quicklisp dist when I chanced upon https://github.com/hemml/OMGlib/blob/main/example.lisp . Thought I'd share.

(ql:quickload :omg)

Maybe you'll find something useful, maybe not. 

Godspeed,

**Comment by rabbibotton** [comment]
Thanks :) I plan on adding server side CL using ECL with WASM but maybe worth looking at this project and parescript to add sooner.

**Comment by rabbibotton** [comment]
I just saw that OMG has a license virus, ie GPL. While I have large base of GPL end user products I have written, it is a crime to place on compilers and developer libraries. If they ever change license let me know.

**Reply by bhw-foss** [reply]
ah that's too bad. Keep the updates CLOG updates coming!
